language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config8.json

```diff
@@ -43,15 +43,15 @@
           "value": ""
         }
       },
-      "sortText": "C",
+      "sortText": "AC",
       "insertText": "TEST_STRING_CONST1",
       "insertTextFormat": "Snippet"
     },
     {
       "label": "listener1",
       "kind": "Variable",
       "detail": "module1:Listener",
-      "sortText": "C",
+      "sortText": "AC",
```

Comment:
Again, we should prioritize string variables and functions that return strings

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39408